src/main/java/org/broadinstitute/consent/http/resources/MetricsResource.java

```diff
@@ -24,19 +24,22 @@ public MetricsResource(MetricsService metricsService) {
   @Path("/dar/decision")
   @Produces(MediaType.TEXT_PLAIN)
   public Response getDarMetricsData() {
-    String joiner = "\t";
-    StringBuilder tsv = new StringBuilder(DarDecisionMetrics.getHeaderRow(joiner));
-    metricsService.generateDarDecisionMetrics().forEach(m -> tsv.append(m.toString(joiner)));
-    return Response.ok(tsv.toString()).build();
+    Type type = Type.DAR;
+    return getMetricsData(type);
```

Comment:
This can be inlined too:
```suggestion
    return getMetricsData(Type.DAR);
```